src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -599,6 +616,8 @@ def run(self) -> None:
             self.scale_and_filter_results = scaled.scale_and_filter_results
             logger.info("Scale and filtering:\n%s", self.scale_and_filter_results)
 
+            logger.notice(banner("Merging"))  # type: ignore
```

Comment:
We have two "Merging" banners - the first for all data, and then this one for filtered data, but no merging banners when scaling and merging clusters (ok it's included in the 'Scaling and merging coordinate clusters' banner). I don't think it makes sense to have a separate merging banner for each coordinate cluster. Therefore, would it be clearer to make these two merging banners unique by changing them to something like "Merging (All data)" and "Merging (Filtered)", which matches the names that appear in the html report?